.yarnrc.yml

```diff
@@ -19,4 +19,8 @@ packageExtensions:
     peerDependencies:
       react-dom: ^16.14.0
 
+plugins:
```

Comment:
I was using `yarn upgrade-interactive` as a simple tool to upgrade some packages for all of our projects. With the new version of `yarn` it's a separate tool. I left that, because I thought that we may make a use of it.

Reply:
OK, I guess it doesn't hurt.